src/Fingerprint.js

```diff
@@ -226,6 +234,8 @@ $.extend( SELF.prototype, {
 					+ 'wb.datamodel.MultiTermMap' );
 			}
 			this._aliases = aliases;
+		} else if ( aliases.isEmpty() ) {
```

Comment:
To me it looks like this must go up, above the previous block. I wonder why no tests catches this?